var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -97,21 +97,24 @@ def _build_mods(self, mods_location, link_flag='', include_flag='',
                                mods_location)
 
         assert os.path.isfile(os.path.join(output_dir, 'special'))
-        return include_flag, link_flag
+        return include_flag_raw, link_flag_raw
+
+    def _nrnivmodlcore_params(self, include_flag, link_flag):
+        return ['-n', self.mech_name, '-i', include_flag, '-l', link_flag, '-V']
```

Comment:
```suggestion
        return ['-n', self.mech_name, '-i', include_flag, '-l', link_flag]
```
seems like -V flag has bug in nrnivmodl-core. If it is used then all mod files are not compiled. (it consumes extra option that result into only partial file compilation)

Reply:
very strange. We have always used `-V` it may be a problem with arg passing then